digdag-server/src/main/java/io/digdag/server/rs/AttemptResource.java

```diff
@@ -315,7 +315,7 @@ public Response startAttempt(RestSessionAttemptRequest request)
                 // If a group error has occurred,
                 // exclude the group's child/dynamically generated tasks from successTasks
                 // even if they are in SUCCESS state
-                .filter(task -> task.getParentId().isPresent() && !groupRetryErrorTaskIds.contains(task.getParentId().get()))
+                .filter(task -> !groupRetryErrorTaskIds.contains(task.getParentId().orNull()))
```

Comment:
Just in case, you want to add the task which has no parent in successTasks, right ?

Reply:
@yoyama Yes, you're right.